Prepare Documentation (#17)
andremw/enotas: README.md

# eNotas
[![Build Status](https://travis-ci.org/andremw/enotas.svg?branch=master)](https://travis-ci.org/andremw/enotas) [![codecov](https://codecov.io/gh/TechnionYP5777/project-name/branch/master/graph/badge.svg)](https://codecov.io/gh/andremw/enotas) [![Maintainability](https://api.codeclimate.com/v1/badges/13fa32cd75932df81150/maintainability)](https://codeclimate.com/github/andremw/enotas/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/13fa32cd75932df81150/test_coverage)](https://codeclimate.com/github/andremw/enotas/test_coverage)
A functional Nodejs Wrapper for eNotas API

# Contributing 

> You should create a fork of this repo, and open a Pull Request suppling our code pattern and contributers guide. You could find issues in our issues tab on github, if you make a improvement (non-issued PR) please create an issue reporting what you want to improve and link to your PR.

Reviewer: ```suggestion
You should create a fork of this repo, and open a Pull Request (according to?) our code pattern and contributors guide. If you want to make an improvement (non-issued PR) please create an issue reporting what you want to improve and link to your PR.
```

Reviewer: I think we can remove "you could find issues..." because we can suppose that the person using GitHub knows where to find the issues.
